docs/framework/network-programming/understanding-webrequest-problems-and-exceptions.md

```diff
@@ -14,15 +14,15 @@ ms.author: "markl"
-|<xref:System.Net.WebExceptionStatus.SecureChannelFailure>|An error occurred in the SSL transaction, or there is a certificate problem.|The .NET Framework version 1.1 only supports SSL version 3.0. If the server is using only TLS version 1.0 or SSL version 2.0, the exception is thrown. Upgrade to .NET Framework version 2.0, and set <xref:System.Net.ServicePointManager.SecurityProtocol%2A> to match the server.<br /><br /> The client certificate was signed by a Certificate Authority (CA) that the server does not trust. Install the CA's certificate on the server. See [http://support.microsoft.com/?id=332077](https://go.microsoft.com/fwlink/?LinkID=179654).<br /><br /> Make sure you have the latest service pack installed.|  
+|<xref:System.Net.WebExceptionStatus.TrustFailure>|The server certificate could not be validated.|Try to open the URI using Internet Explorer. Resolve any Security Alerts displayed by IE. If you cannot resolve the security alert, then you can create a certificate policy class that implements <xref:System.Net.ICertificatePolicy> that returns `true`, and pass it to <xref:System.Net.ServicePointManager.CertificatePolicy%2A>.<br /><br /> Refer to [https://support.microsoft.com/?id=823177](https://go.microsoft.com/fwlink/?LinkID=179653).<br /><br /> Make sure that the certificate of the Certificate Authority that signed the server certificate is added to the Trusted Certificate Authority list in Internet Explorer.<br /><br /> Make sure that the host name in the URL matches the common name on the server certificate.|  
+|<xref:System.Net.WebExceptionStatus.SecureChannelFailure>|An error occurred in the SSL transaction, or there is a certificate problem.|The .NET Framework version 1.1 only supports SSL version 3.0. If the server is using only TLS version 1.0 or SSL version 2.0, the exception is thrown. Upgrade to .NET Framework version 2.0, and set <xref:System.Net.ServicePointManager.SecurityProtocol%2A> to match the server.<br /><br /> The client certificate was signed by a Certificate Authority (CA) that the server does not trust. Install the CA's certificate on the server. See [https://support.microsoft.com/?id=332077](https://go.microsoft.com/fwlink/?LinkID=179654).<br /><br /> Make sure you have the latest service pack installed.|  
```

Comment:
same for all other support links on this page. Change the link to the \<> format and use the support link